services/jetbrains/jetbrains-downloads.service.js

```diff
@@ -29,15 +30,17 @@ export default class JetbrainsDownloads extends JetbrainsBase {
     pattern: ':pluginId',
   }
 
-  static examples = [
-    {
-      title: 'JetBrains plugins',
-      namedParams: {
-        pluginId: '1347',
+  static openApi = {
+    '/jetbrains/plugin/d/{pluginId}': {
+      get: {
+        summary: 'JetBrains plugin Downloads',
```

Comment:
```suggestion
        summary: 'JetBrains Plugin Downloads',
```

For consistency with the other JetBrains badge :)